src/pds_doi_service/core/actions/roundup.py

```diff
@@ -0,0 +1,124 @@
+#
+#  Copyright 2022, by the California Institute of Technology.  ALL RIGHTS
+#  RESERVED. United States Government Sponsorship acknowledged. Any commercial
+#  use must be negotiated with the Office of Technology Transfer at the
+#  California Institute of Technology.
+#
+"""
+======
+roundup.py
+======
+
+Contains functions for sending email notifications for recently-updated DOIs.
+"""
+import json
+import logging
+import os
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+from email import encoders
+from email.mime.base import MIMEBase
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from typing import Dict
+from typing import List
+
+import jinja2
+from pds_doi_service.core.db.doi_database import DOIDataBase
+from pds_doi_service.core.entities.doi import DoiRecord
+from pds_doi_service.core.util.emailer import Emailer as PDSEmailer
+from pkg_resources import resource_filename
+
+
+def get_start_of_local_week() -> datetime:
+    """Return the start of the local-timezones week as a tz-aware datetime"""
+    today = datetime.now().date()
```

Comment:
Do we want to root ourselves to UTC here with `datetime.datetime.now(datetime.timezone.utc)`?

Reply:
_Possibly_.  @jordanpadams ?

Current behaviour is to return data for the preceding Mon-Sun week, referenced to the tz of the host on which doi-service is running.

Your judgement call.

Reply:
@nutjob4life @alexdunnjpl doesn't matter. weekly on whatever cadence is fine.